src/main/kotlin/racingcar/domain/car/Car.kt

```diff
@@ -1,29 +1,35 @@
 package racingcar.domain.car
 
+import racingcar.domain.car.carengine.CarEngine
+import racingcar.domain.car.carengine.RandomCarEngine
 import racingcar.domain.record.CarRecord
 
 class Car(
-    val lineNumber: Int,
-    private val carEngine: CarEngine = RandomCarEngine
+    val name: String,
+    private val carEngine: CarEngine = RandomCarEngine,
+    position: Int = START_POSITION
 ) {
-    var position: Int = START_POSITION
+    var position: Int = position
         private set
+    var carRecord = CarRecord(listOf())
+        private set
+
+    init {
+        require(name.length <= 5) { NAME_LIMIT_ERROR_MESSAGE }
```

Comment:
```suggestion
        require(name.length <= NAME_LENGTH_LIMIT) { NAME_LIMIT_ERROR_MESSAGE }
```
정의하신 상수값을 활용해보면 어떨까요?

Reply:
변경했습니다!